aquarius/app/dao.py

```diff
@@ -84,7 +85,7 @@ def is_listed(services):
                 ):
                     return service["attributes"]["curation"]["isListed"]
 
-    def run_es_query(self, data):
+    def run_es_query(self, data, with_metadata=False):
```

Comment:
would be nice to have this option documented. Also update the docstr to reflect the new return types.